perma-payments/config/urls.py

```diff
@@ -0,0 +1,27 @@
+"""Top-level URL Configuration
+
```

Comment:
why is this file necessary?

Reply:
This is automatically created by Django when you pop up a new project, and, if you are following along with the docs/Quick Start, they have you create a urls.py inside your app, even if you are only using one. I don't have strong feelings about it, but since Perma and Capstone work this way too, and since it seems to be the default in Django land, my instincts are to stick with it, even if it isn't strictly necessary for this project at this time. 

The separation seems good to me on the level of taste: there's something awfully nice about having the urls and views for the perma payments app all in the same folder (=status quo).

Would you prefer to see all the routes for an application in a top-level folder, as a matter of taste? I think that's how Rails works, if I'm not mistaken....

Reply:
Yeah! I was totally reminded of all the Django weirdness a couple days later when I ran `django-admin startproject` and `django-admin startapp`. How quickly we forget. 
I don't have strong feelings either way — a more flat structure makes sense to me aesthetically if we know for a fact we don't expect the project to contain any more apps, but if this is what Django wants, then maybe this is what Django gets